test/testing/test_wigner.jl

```diff
@@ -0,0 +1,32 @@
+
+@testset "TestWigner" begin
+
+##
+using ACE, StaticArrays, ACE.SphericalHarmonics;
+using ACEbase
+using ACE.SphericalHarmonics: index_y;
+using ACE: evaluate
+using Random, Printf, Test, ACE.Testing
+using ACE, Test, Printf, LinearAlgebra, BenchmarkTools
+
+##
+
+
+@info("Check correctness of Wigner-D matrices")
+
+L = 1
```

Comment:
Can you please generalise this test to allow `for L = 0:3` as in the other test?